package.json

```diff
@@ -9,6 +9,7 @@
     "clean": "gatsby clean",
     "author": "LIMIT_BLOG_PAGES=1 SKIP_DOCS=true gatsby develop",
     "build": "gatsby build",
+    "build:incremental": "GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES=true gatsby build",
```

Comment:
Used Gatsby's incremental build flag

Reply:
Q: what does this key mean? why don't we use it in the regular builds?

Reply:
It looks like Gatsby has [this feature on by default](https://www.gatsbyjs.com/docs/reference/release-notes/v3.0/#incremental-builds-in-oss). So, caching `public` and `.cache` should be enough.

Reply:
This was a Gatsby 3 flag for a feature that became default in Gatsby 4, so it can be removed here.